app/models/user_authorization_document.rb

```diff
@@ -0,0 +1,21 @@
+class UserAuthorizationDocument < AuthorizationDocument
+  OPTIONAL_DOCUMENTS = [
+    :certificates,
+    :last_year_activities_report,
+    :organization_current_plan
+  ]
+
+  def attachment_present
+    assign_document_error(category_i18n) if invalid_document?
+  end
+
+  def invalid_document?
+    return false if optional_document?
+
+    attachment.url.blank?
+  end
```

Comment:
I think it could be easier to read in this way:
```Ruby
def invalid_document?
  !optional_document? && attachment.url.blank?
end

def optional_document?
  OPTIONAL_DOCUMENTS.include?(category.to_sym)
end
```

Or if you just use the `optional_document?` here, maybe it could go like
```Ruby
def invalid_document?
  document_required? && attachment.url.blank?
end

def document_required?
  OPTIONAL_DOCUMENTS.exclude?(category.to_sym)
end
```